backend-shared/exportfuncs.cpp

```diff
@@ -30,6 +32,28 @@ bool ExportCallback::canceled() const
 }
 
 #if !defined(SUBSURFACE_MOBILE)
+
+#include "desktop-widgets/mainwindow.h" // Currently needed for profile printing. TODO: remove.
+
+static void exportProfile(const struct dive *dive, const QString filename)
+{
+	ProfileWidget2 *profile = MainWindow::instance()->graphics;
+	profile->setToolTipVisibile(false);
+	profile->setPrintMode(true);
+	double scale = profile->getFontPrintScale();
+	profile->setFontPrintScale(4 * scale);
+	profile->plotDive(dive, 0, false, true);
+	QImage image = QImage(profile->size() * 4, QImage::Format_RGB32);
+	QPainter paint;
+	paint.begin(&image);
+	profile->render(&paint);
+	image.save(filename);
+	profile->setToolTipVisibile(true);
+	profile->setFontPrintScale(scale);
+	profile->setPrintMode(false);
+	profile->plotDive(dive, 0, true); // TODO: Shouldn't this plot the current dive?
```

Comment:
well, it plots what you pass in, right?

Reply:
This is copy&paste of a comment introduced in 2789bb05b133a7cf54081d58d4f5c51c8977e951

I'm not sure what I was thinking at that time...?

In any case, as you note, the bogus comment will be removed later on.